app/views/nsm/additional_fees/_adjusted.html.erb

```diff
@@ -1,6 +1,28 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <p><%= t('.no_data') %></p>
+    <% if records.rows.none? { _1.any_adjustments? } %>
+      <p><%= t('.no_data') %></p>
+    <% else %>
+      <%=
+      caption = t('.table_caption')
+      head = Nsm::V1::AdditionalFee.headers
+      rows = records.rows.reject{|item| item.type == :total}.map do |additional_fee|
+        fields = additional_fee.table_fields.dup
```

Comment:
I don't know if this needs a check to only show rows that actually have adjustments

Reply:
done